### ISSUE TITLE
PPU: make it more clear (or change) £1000 filter

### ISSUE THREAD

**Issue by HelenCEBM** [ebmdatalab/openprescribing#1606]
It's not written on the main PPU pages and may cause confusion. 

Could we consider a different filter eg. cost per list size? Some small CCGs have very few savings listed and may be interested to see more e.g. Erewash CCG has only 2 savings listed for January. https://openprescribing.net/ccg/03X/price_per_unit/

**Comment by sebbacon** [comment]
Specifically, we could ask some smaller and larger CCGs:

* Currently, we discard savings at a CCG level that are less than £1000. What is the minimum saving you would consider worth attempting to make at a CCG level? 

Initially, we could compare the opinions of Erewash, Hardwick (both small), and North Derbyshire and Southern Derbyshire (both large).

**Comment by HelenCEBM** [comment]
As a relatively quick fix for this, can we just:
- [ ] optional - reduce the filter to say £500? It won't make much difference to large CCGs as they will just be on later pages of their results, though they might notice their total savings opportunities increase;
- [ ] add a note to say that there's a £1000 (or £500) filter in place, and to contact us if you're interested in smaller savings?

Any thoughts @richiecroker @brianmackenna ?

**Comment by HelenCEBM** [comment]
Suggest amend this
> We have identified about £xxx of possible savings in CCG/practice in [month].

to say:
> We have identified about £xxx of possible savings in CCG/practice in [month], summed from individual drug savings of >=£1000 / £50 each. If you would like to see more savings, contact us[link]. 


Other options are to hide this in the "show more" text (probably not looked at often) where we currently say we show the "biggest" savings, and/or put in the FAQs.

Other suggestions?

**Comment by sebbacon** [comment]
I think inverting the wording is clearer: 
> We have identified about £xxx of possible savings in CCG/practice in [month], but excluded individual drug savings of < £1000 / £50 each. If you would like to see these smaller savings too, contact us[link].

**Comment by HelenCEBM** [comment]
Good idea - I would try to make it short to increase likelihood of anyone reading it, e.g.
> We have identified about £xxx of possible savings in CCG/practice in [month]. We have excluded small individual drug savings (< £1000 / £50), but if you would like to see more, contact us[link].

**Comment by richiecroker** [comment]
We have now reduced this down to <£200 / £10.  This should pick up most of the PPUs, without lots of tiny amounts.
